src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Only

    #define NOTIFICATION_WAIT_MODE false

as it seems.

Remember this is a merge. Not all modifications are mine ...